### PR TITLE
Make Camera Settings Center Point

### PR DESCRIPTION
**Pull request by Ubaydulloh99**
Camera Window Center Point component - done

image

https://drive.google.com/file/d/1dFIg_LrZdNT0oAMcE4853zBOVPCCvQPL/view?usp=sharing

usage video

### CODE REVIEW

File: qgcimages.qrc [hunk 1]
```diff
@@ -196,5 +196,11 @@
         <file alias="Yield.svg">src/ui/toolbar/Images/Yield.svg</file>
         <file alias="ZoomMinus.svg">src/FlightMap/Images/ZoomMinus.svg</file>
         <file alias="ZoomPlus.svg">src/FlightMap/Images/ZoomPlus.svg</file>
+        <file alias="CenterPoint_Square.svg">custom/res/Images/Camera Setting_Center Point_Square(With).svg</file>
```
Comment (thanhkaist) [inline qgcimages.qrc]: Change the file name to Camel case. e.g CameraSettingCenterPointSquare.svg

File: qgcimages.qrc [hunk 2]
```diff
@@ -196,5 +196,11 @@
...
+        <file alias="Circle_WithCenter.svg">custom/res/Images/Circle(With Center Point).svg</file>
```
Comment (thanhkaist) [inline qgcimages.qrc]: Change the file name to Camel case

File: qgcimages.qrc [hunk 3]
```diff
@@ -196,5 +196,11 @@
...
+        <file alias="Circle_WC.svg">custom/res/Images/Circle(Without Center Point).svg</file>
```
Comment (thanhkaist) [inline qgcimages.qrc]: Change the file name to Camel case

File: src/QmlControls/CenterPoint.qml [hunk 4]
```diff
@@ -0,0 +1,182 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.14
+import QtPositioning 5.14
+
+Rectangle {
+    id: root
+    color: '#000000'   // black
+
+    property var rootWidth: null;
+    property var rootHeight: null;
```
Comment (thanhkaist) [inline src/QmlControls/CenterPoint.qml]: property int rootWidth: Screen.width
property int rootHeight: Screen.height

File: src/QmlControls/CenterPoint.qml [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+
+    width: rootWidth / 3.24
+    height: rootHeight / 1
+    Text {
+        id: centerPointText
+        color: "#FFFFFF"    // white
```
Comment (thanhkaist) [inline src/QmlControls/CenterPoint.qml]: define color property in root item for this color

File: src/QmlControls/CenterPoint.qml [hunk 6]
```diff
@@ -0,0 +1,182 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.14
+import QtPositioning 5.14
+
```
Comment (thanhkaist) [inline src/QmlControls/CenterPoint.qml:6]: import ScreenTool

**Review by thanhkaist — COMMENTED** [review]
Check the proposal

File: src/QmlControls/CenterPoint.qml (hunk 6 above)
Comment (Ubaydulloh99) [reply src/QmlControls/CenterPoint.qml:6]: done

File: src/QmlControls/CenterPoint.qml (hunk 4 above)
Comment (Ubaydulloh99) [reply src/QmlControls/CenterPoint.qml]: done

File: src/QmlControls/CenterPoint.qml (hunk 5 above)
Comment (Ubaydulloh99) [reply src/QmlControls/CenterPoint.qml]: done

File: src/QmlControls/CenterPoint.qml [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+        anchors.topMargin: 20 // Editable margin between text1 and column1
+        anchors.left: parent.left
+        anchors.right: parent.right
+        anchors.bottom: parent.bottom
+
+        spacing: 5
+
+        ItemDelegate {
+            width: parent.width
+            height: 52
```
Comment (thanhkaist) [inline src/QmlControls/CenterPoint.qml]: Hardcoding. Please use property binding

File: src/QmlControls/CenterPoint.qml [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+                text: 'None'
+                font.pixelSize: ScreenTools.mediumFontPointSize * 2
+                color: white
+                anchors.verticalCenter: parent.verticalCenter
+            }
+        }
+
+        ItemDelegate {
+            width: parent.width
+            height: 52
```
Comment (thanhkaist) [inline src/QmlControls/CenterPoint.qml]: Hardcoding. Please use property binding

File: src/QmlControls/CenterPoint.qml [hunk 9]
```diff
@@ -0,0 +1,178 @@
...
+                color: white
+                text: qsTr("Square(With Center Point)")
+                font.pixelSize: ScreenTools.mediumFontPointSize * 2
+                anchors.verticalCenter: parent.verticalCenter
+            }
+        }
+
+        ItemDelegate {
+            width: parent.width
+            height: 52
```
Comment (thanhkaist) [inline src/QmlControls/CenterPoint.qml]: Hardcoding. Please use property binding

File: src/QmlControls/CenterPoint.qml [hunk 10]
```diff
@@ -0,0 +1,178 @@
...
+                text: qsTr("Cross")
+                font.pixelSize: ScreenTools.mediumFontPointSize * 2
+                anchors.verticalCenter: parent.verticalCenter
+            }
+        }
+
+        ItemDelegate {
+            id: itemDelegate4
+            width: parent.width
+            height: 52
```
Comment (thanhkaist) [inline src/QmlControls/CenterPoint.qml]: Hardcoding. Please use property binding

File: src/QmlControls/CenterPoint.qml [hunk 11]
```diff
@@ -0,0 +1,178 @@
...
+                color: white
+                text: qsTr("Circle(Without Center Point)")
+                font.pixelSize: ScreenTools.mediumFontPointSize * 2
+                anchors.verticalCenter: parent.verticalCenter
+            }
+        }
+
+        ItemDelegate {
+            width: parent.width
+            height: 52
```
Comment (thanhkaist) [inline src/QmlControls/CenterPoint.qml]: Hardcoding. Please use property binding

**Review by thanhkaist — APPROVED** [review]

**Pull request merged**